packages/devextreme/js/__internal/grids/data_grid/m_widget_base.ts

```diff
@@ -139,6 +136,7 @@ class DataGrid extends GridCoreWidget<Properties> {
   }
 
   protected _setDeprecatedOptions() {
+    // @ts-expect-error
```

Comment:
What changed here that we need the ```@ts-expect-error``` now?
Let's add the comment if we really need it here :)

Reply:
I removed `_setDeprecatedOptions` in `gridCore` base class, now super calls dxWidget's `_setDeprecatedOptions`. It isn't described properly in d.ts. I won't mark it additionaly, as we don't mark same situations usually